build_tools/cmake/build_docs.sh

```diff
@@ -58,3 +59,61 @@ scripts/update_op_coverage.py ${BUILD_DIR}
 # Update e2e_coverage.md
 PYTHON_BIN=`which python3` scripts/update_e2e_coverage.py ${BUILD_DIR}
 
+# Generate IR conversion flow examples
+
+# The list of input IR examples
+declare -a ir_examples=(
+  iree/samples/ops/mhlo-dot.mlir
+  iree/samples/ops/mhlo-conv.mlir
```

Comment:
Maybe drop conv for now. THere are still some things that need to be cleaned up there.. I think we can use an example of dot, and elementwise operations that get fused (i.e. the ones used in the codegen documentation.

Reply:
Okay. I'll remove conv for now then. Basically wanting to set up the flow here; feel free to put up more examples you'd like to generate! :)